common/src/main/java/gov/cms/ab2d/common/service/CacheServiceImpl.java

```diff
@@ -64,6 +76,19 @@ private Integer getMonth(ClearCoverageCacheRequest request) {
         return month;
     }
 
+    private Integer getYear(ClearCoverageCacheRequest request) {
+        final Integer year = request.getYear();
+
+        final int currentYear = OffsetDateTime.now(ZoneOffset.UTC).getYear();
```

Comment:
Others can confirm but we are using Eastern time as the standard.

Reply:
For storing HPMS data we've been using EST, other data is stored as UTC, which is pretty standard

Reply:
@adaykin yes but I want to make sure I always have the maximum year it could be so at 12:00 AM January 1st, 2021 UTC it is still 2020 EST which could cause a problem if someone requests a job for January. I don't know if it matters but it seemed safest.